### ISSUE TITLE
tapViewWithAccessibilityLabel passes, but takes nearly a minute

### ISSUE THREAD

**Issue by tempire** [kif-framework/KIF#561]
This test runs:

```
func testSidebarCompetitors() {
    let t = tester()
    println(NSDate())

    t.swipeViewWithAccessibilityLabel("revealControllerView", inDirection: .Right)
    println(NSDate())

    t.waitForAnimationsToFinish()
    println(NSDate())

    t.tapViewWithAccessibilityLabel("SidebarCompetitors")
    println(NSDate())
}
```

The log output:

```
Test Case '-[PointsTests.TestOne testSidebarCompetitors]' started.
2015-02-02 08:57:02 +0000
2015-02-02 08:57:02 +0000
2015-02-02 08:57:03 +0000
2015-02-02 08:57:56 +0000
Test Case '-[PointsTests.TestOne testSidebarCompetitors]' passed (61.397 seconds).
```

The swipe and wait tests run in an acceptable amount of time, but the tapViewWithAccessibility takes nearly a full minute before it passes. The label exists, as the test would otherwise time out and fail after 10 seconds.

**Comment by phatmann** [comment]
Can you trace through the code and see what is causing the delay? KIF should timeout after 10 seconds.

**Comment by phatmann** [comment]
Is this a dup of #464?

**Comment by phatmann** [comment]
I do not think this is a dup of #464, now that we have found the root cause and fixed it (see #605).
